configs/experiment_schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Experiment schema",
+  "type": "object",
+  "minProperties": 1,
```

Comment:
This seems to allow an experiment to have just a single `description` without any `rounds`. `rounds` is required. It seems we may need this: https://json-schema.org/understanding-json-schema/reference/object.html#required-properties